instrumentation/logback/logback-1.0.0/library/NOTICE.txt

```diff
@@ -0,0 +1,19 @@
+This product contains a modified part of Armeria, distributed by LINE:
```

Comment:
I would generally copy the license header into the file I copied as well, but spotless deletes the second header if I try. For now I've added this NOTICE but let me know if there's something I should do here.